# Gait Phase
Header header
# GAIT PHASE:
# 1: Heel Strike
# 2: Flat foot
# 3: Mid-stance
# 4: Heel-off
# 5: Toe-off
# 6: Mid-swing
# 7: Invalid state!
int16 phase
